components/Slider.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from 'react-responsive-carousel';
+
+export default class Hazhar extends Component {
+	render() {
+		return (
+			<div className='my-24'>
+                <h2 className='text-center text-xl font-bold'> مشتریان ما </h2>
```

Comment:
- Install recommended extensions mentioned in ```rekar-accounting/docs``` repo in your **vscode** to have a better development experience and enable code formatting.